packages/harmony/src/components/input/SelectablePill/SelectablePill.tsx

```diff
@@ -57,6 +80,11 @@ export const SelectablePill = (props: SelectablePillProps) => {
       paddingInline: theme.spacing.l,
       boxShadow: theme.shadows.near
     }),
+    ...(size === 'oversized' && {
```

Comment:
adding styles to selectable pill to implement radio-pill. using `size === 'oversized' as the clue to do this, since it basically is just a really large selectable pill